### PR TITLE
Issue 10545: Convert complex zmg BBCode elements

### LINKED ISSUE #10545: Tag [zmg] of ZAP
When a friend on zap posts an image I don't see it but I see the tag (for example):

`[zmg width="1024" height="682" alt="TXT ALT"]https://venera.social/photo/preview/XXX/XXXXX[/zmg]`

I think the tag is not interpreted.

**Version**
This is Friendica, version 2021.07 that is running at the web location https://venera.social. The database version is 1424/1424, the post update version is 1400/1400.

### PR DESCRIPTION
**Pull request by annando**
Fixes #10545

### CODE REVIEW

**Review by MrPetovan — CHANGES_REQUESTED** [review]
Please increase the BBCode version constant!

File: src/Content/Text/BBCode.php [hunk 1]
```diff
@@ -1067,6 +1067,43 @@ function ($match) use ($callback, $uriid) {
...
+	private static function convertImages(string $text, int $simplehtml, int $uriid = 0):string
+	{
+		DI::profiler()->startRecording('rendering');
+		$return = preg_replace_callback(
+			"/\[[zi]mg(.*?)\]([^\[\]]*)\[\/[zi]mg\]/ism",
+			function ($match) use ($simplehtml, $uriid) {
+				$attribute_string = $match[1];
+				$attributes = [];
+				foreach (['alt', 'width', 'height'] as $field) {
+					preg_match("/$field=(['\"])(.+?)\\1/ism", $attribute_string, $matches);
```
Comment (MrPetovan) [inline src/Content/Text/BBCode.php:1087]: This covers `field="value"` and `field='value'`, but do we know if `field=value` exists as well?

Comment (annando) [reply src/Content/Text/BBCode.php:1087]: I currently don't find the Zap repository. But yesterday I had a quick look into it. AFAIK there are always quotes.

**Comment by annando** [comment]
@MrPetovan version is increased.

**Review by MrPetovan — APPROVED** [review]

**Pull request merged**